proxies/go/handler_client.go

```diff
@@ -59,10 +59,12 @@ func clientHandler(w http.ResponseWriter, r *http.Request) {
 		EventFlushIntervalMS:    time.Duration(reqBody.Options.EventFlushIntervalMS * 1000000),
 		EnableCloudBucketing:    reqBody.EnableCloudBucketing,
 		OnInitializedChannel:    onInitializedChannel,
+		// TODO: Properly implement SSE tests. Currently the init delay on config pull breaks tests.
+		DisableServerSentEvents: true,
```

Comment:
this field should be named `disableRealtimeUpdates` to match the other SDKs

Reply:
We aren't calling it realtime updates for server SDKs as far as i know - it's being called SSE as far as I know? @jonathannorris 

Realtime updates is the client sdk concept

Reply:
Yeah, it's the same thing, it should be named the same.

Reply:
kk - i'll update the SDK then

Reply:
https://docs.devcycle.com/sdk/client-side-sdks/javascript/javascript-gettingstarted#initialization-options

Reply:
SSE is the name of a technology which powers the "realtime updates" feature, SSE by itself would be a meaningless term to most people I think